butane_core/src/migrations/migration.rs

```diff
@@ -86,8 +86,19 @@ pub trait MigrationMut: Migration {
     /// Adds an abstract table to the migration. The table state should
     /// represent the expected state after the migration has been
     /// applied. It is expected that all tables will be added to the
-    /// migration in this fashion.
-    fn write_table(&mut self, table: &ATable) -> Result<()>;
+    /// migration in this fashion, if they were modified in this migration.
+    fn add_modified_table(&mut self, table: &ATable) -> Result<()>;
```

Comment:
I think I would prefer not to rename this method. I'm open to the argument that the new name is better/clearer but I still prefer not to break backwards compatibility solely to clarify a method name.

Reply:
IMO now is the time to do any breaking changes.

We've already broken it with the removal of `.state`, and so far there is only one dependent crate https://github.com/DimmKG/butane_rocket_pool per https://crates.io/crates/butane/reverse_dependencies , and one other project https://github.com/TmLev/crud-grpc-rust-tonic-butane-postgres using it per https://github.com/search?q=butane+language%3Arust&type=repositories .  Code search reveals one more : https://github.com/janosimas/code_as_db , and I think I found another one a year ago, but cant find it now.

Reply:
Public OSS doesn't necessarily tell they true number of dependents, but you're certainly right that it's not high. 
To my mind this is one of the less necessary breaking changes, but on the other hand the chances that it even breaks any dependents is very slim, since it's primarily for use of butane itself and not needed by ordinary projects.
So I won't block it going forward.